gspread/worksheet.py

```diff
@@ -3201,3 +3204,88 @@ def cut_range(
         }
 
         return self.client.batch_update(self.spreadsheet_id, body)
+
+    def add_validation(
+        self,
+        source: str,
```

Comment:
could you please rename the argument `source` to `range`, we intend to use `range` for range name.

Reply:
I get lint errors when I do that.

Reply:
I think we missunderstood each others, what I mean is: the method argument currently name `source` should be renamed `range` please.
This is a method argument name I don't see how it can fail the linter :thinking:  unless there is something I don't see here.